### ISSUE TITLE
Fine-tuning Details on CC12M and CC12M-NegFull?

### ISSUE THREAD

**Issue by MiaoKYJMSZD** [m1k2zoo/negbench#8]
Thanks for your nice work!

 Could you please share the batch size, learning rate, and weight decay settings used when fine-tuning CLIP and NegCLIP on CC12M and CC12M-NegFull? Thank you!
